### PR TITLE
fix issue#519 - Create Page

### PR DESCRIPTION
**Pull request by binbin1802**
image
image

### CODE REVIEW

File: packages/app-lixi/src/styles/style.less [hunk 1]
```diff
@@ -791,7 +817,7 @@ code {
     }
   }
 
-  .ant-popover-arrow > .ant-popover-arrow-content::before {
+  .ant-popover-arrow>.ant-popover-arrow-content::before {
```
Comment (vince8x) [inline packages/app-lixi/src/styles/style.less:820]: It seems the code should have space between element and ">"?

File: packages/app-lixi/src/styles/style.less [hunk 2]
```diff
@@ -546,7 +556,7 @@ code {
       .ant-select {
         border: 2px solid var(--border-color-base);
         background: #fff;
-        padding: 9px 1rem;
+        margin: 0 0 0 0;
```
Comment (eric-son12) [inline packages/app-lixi/src/styles/style.less:559]: should to use: margin: 0;

File: packages/app-lixi/src/styles/style.less [hunk 3]
```diff
@@ -588,7 +598,23 @@ code {
           background: #fff;
           text-align: left;
           border: 0;
-          padding: 0;
+          padding: none;
```
Comment (eric-son12) [inline packages/app-lixi/src/styles/style.less:601]: Not use padding: none

File: packages/app-lixi/src/styles/style.less (hunk 2 above)
Comment (binbin1802) [reply packages/app-lixi/src/styles/style.less:559]: okay, i fix now

File: packages/app-lixi/src/styles/style.less (hunk 1 above)
Comment (binbin1802) [reply packages/app-lixi/src/styles/style.less:820]: It's always rollback without spacing between elements when I save ( Ctrl + S )

**Review by vince8x — APPROVED** [review]

**Pull request merged**